### PR TITLE
Add HTML, CSS, Linters, README.md, Image

### PR DESCRIPTION
**Pull request by Akhtar-Joomun**
Project Portfolio: setup and mobile version skeleton
It's about a project of building a website for my Portfolio.

Files Add:
- HTML
- CSS
- README.md
- Linters
- Image
- A screenshot

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                    </svg>
+                </a>
+            </li>
+            <li class="list-item">
+                <a href="#">
+                    <svg width="20" height="16" viewBox="0 0 20 16" xmlns="http://www.w3.org/2000/svg">
+                        <path
+                            d="M18.4042 1.88042L20 0.337684V0H14.4717L10.5317 9.90989L6.04917 0H0.2525V0.337684L2.11667 2.60547C2.29833 2.77305 2.39333 3.01811 2.36917 3.26484V12.1768C2.42667 12.4977 2.32333 12.8278 2.1 13.0611L0 15.6328V15.9663H5.95417V15.6286L3.85417 13.0611C3.62667 12.8269 3.51917 12.5027 3.565 12.1768V4.46821L8.79167 15.9705H9.39917L13.8933 4.46821V13.6312C13.8933 13.8728 13.8933 13.9225 13.7367 14.0808L12.12 15.6615V16H19.9633V15.6623L18.405 14.1204C18.2683 14.016 18.1975 13.8408 18.2267 13.6707V2.33011C18.1975 2.15916 18.2675 1.984 18.4042 1.88042Z" />
```
Comment (DeliceLydia) [inline index.html]: - [x] This container is supposed to use a `section` tag for better use of semantic HTML
- [x] Stick to the margins (top, bottom, left, right) defined in the design

File: index.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+                        <path fill-rule="evenodd" clip-rule="evenodd"
+                            d="M10.368,0h102.144c5.703,0,10.367,4.665,10.367,10.367v0 c0,5.702-4.664,10.368-10.367,10.368H10.368C4.666,20.735,0,16.07,0,10.368v0C0,4.665,4.666,0,10.368,0L10.368,0z M10.368,82.875 h102.144c5.703,0,10.367,4.665,10.367,10.367l0,0c0,5.702-4.664,10.367-10.367,10.367H10.368C4.666,103.609,0,98.944,0,93.242l0,0 C0,87.54,4.666,82.875,10.368,82.875L10.368,82.875z M10.368,41.438h102.144c5.703,0,10.367,4.665,10.367,10.367l0,0 c0,5.702-4.664,10.368-10.367,10.368H10.368C4.666,62.173,0,57.507,0,51.805l0,0C0,46.103,4.666,41.438,10.368,41.438 L10.368,41.438z" />
+                    </g>
+                </svg>
+            </a>
+        </nav>
+    </header>
+    <main>
+        <h1>Hey There. I’m Akhtar</h1>
+        <p class="description">I’m a Software Developer</p>
```
Comment (DeliceLydia) [inline index.html]: - [x] This subtitle is supposed to use a `hx` tag kindly fix that

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Ak0706✋, 

Good job so far! However, there are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Try to commit as much as you can, currently you have only 2commits which are not enough for this project, it is a good practice to commit any meaningful change in your project.


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi @Ak0706 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Akhtar-Joomun** [comment]
Thank you guys for the review!

**Pull request merged**